fixtures/schema/04_geocris/02_fbf/00_tables/00_admin_global_key_mapping.sql

```diff
@@ -0,0 +1,20 @@
+create table if not exists admin_boundary_global_key_mapping (
+    id bigint not null,
+    id_mapping bigint,
+    key character varying,
+    partition character varying,
+    partition_level int,
+    constraint admin_boundary_global_key_mapping_pkey primary key (id_mapping, key)
```

Comment:
`key` is in the form `{schema_name}:{administrative level}:{id_from_origin_table}`.
For example, if we got country name (admin level 0) from `dma` schema (Dominica), the unique `key` is `dma:0:1`.
This format is taken from @dimasciput original approach.

`id_mapping` is for the id we assigned. It should be autoincrement decided from Django, because it needs to be unique in the same partition level.

`partition`, the partition for the mapping. This is with respect to the admin level, which is country, district, sub_district, etc. We currently use standard name from original FBA.

`partition_level`, the admin level. 0 for country, 1 for district, and so on. In the same partition level, all the `id_mapping` must be unique. Because it corresponds to the administrative code assigned in the administrative table `country`, `district`, etc